src/data-workspace/data-details-sidebar/basic-information-follow-up-button.js

```diff
@@ -1,17 +1,15 @@
 import i18n from '@dhis2/d2-i18n'
 import { Button, colors, IconFlag24, Tooltip } from '@dhis2/ui'
 import React from 'react'
-import { useHighlightedField } from '../../shared/index.js'
 import { useSetDataValueMutation } from '../data-value-mutations/data-value-mutations.js'
 import ItemPropType from './item-prop-type.js'
 
 const FollowUpButton = ({ item }) => {
-    const { value } = useHighlightedField()
-    const isEmptyField = !value
+    const isEmptyField = !item?.value
```

Comment:
I changed this as `item` and `value` were the same object. `useHighlightedField()` is used in \<DataDetailsSidebar \/\> and it passes its item down through props

https://github.com/dhis2/data-entry-app/blob/d2745e7dfd83a62d04ec7a0399a3c2c3a7619a69/src/data-workspace/data-details-sidebar/data-details-sidebar.js#L16-L34